Dashboard/management/commands/CreateInviteTokens.py

```diff
@@ -85,9 +85,7 @@ def handle(self, *args, **options):
             new_group.save()
 
         if number_of_tokens < 1 or number_of_tokens > 50:
-            _msg = '{0}Specified number of tokens is insane: {1}'.format(
-                WARNING_MSG, number_of_tokens
-            )
+            _msg = f'{WARNING_MSG}Specified number of tokens is insane: {number_of_tokens}'
```

Comment:
I'm not a fan of the line length limit of only 88 lines, but it's always good to have some auto formatting even if not perfect, because it makes so many things easier. Let's remember to run Black, see commands in `Makefile` that I added today.